app/graphql/types/charges/properties.rb

```diff
@@ -9,6 +9,9 @@ class Properties < Types::BaseObject
       # NOTE: Graduated charge model
       field :graduated_ranges, [Types::Charges::GraduatedRange], null: true, hash_key: :graduated_ranges
 
+      # NOTE: Graduated percentage modle
+      field :graduated_percentage_ranges, [Types::Charges::GraduatedPercentageRange], null: true, hash_key: :graduated_percentage_ranges
```

Comment:
Layout/LineLength: Line is too long. [136/120] (https://rubystyle.guide#max-line-length)